src/Console.cc

```diff
@@ -261,8 +261,8 @@ void FileLogger::Init(const std::string &_directory,
   {
     if (!env(IGN_HOMEDIR, logPath))
     {
-      ignerr << "Missing HOME environment variable."
-        << "No log file will be generated.";
+      std::cerr << "Missing HOME environment variable."
```

Comment:
space between the two messages or new line?

```suggestion
      std::cerr << "Missing HOME environment variable. "
```